src/styles/partials/components/_walking-trails.scss

```diff
@@ -269,11 +273,19 @@
       margin: $default-margin;
       width: 100%;
     }
+
+    .search-container-trails {
+      & > div {
```

Comment:
I went ahead and added a class to the div, so it's not just blank markup. It seemed like a bit of a cleaner solution to just add a class rather than rework styles, some of which have already been minified and added to dev.

Previous styles:
.search-container-trails > div {
    align-items: center;
    display: flex;
    justify-content: center;
    padding: 0 10px;
    width: 100%;
}

@media screen and (max-width: 768px) {
    .search-container-trails > div  {width:100%; padding: 0 20px;}
}

Here's the latest commit:
https://github.com/baltimorecounty/dotgov-components/pull/406/commits/ef85c8a0d6d4301a747f003a855d408bafe80b12